src/domain/entities/figma-team.ts

```diff
@@ -6,7 +6,7 @@ export enum FigmaTeamAuthStatus {
 }
 
 export class FigmaTeam {
-	private readonly _adminInfo: ConnectUserInfo;
+	readonly #adminInfo: ConnectUserInfo;
```

Comment:
We made this change to prevent `_adminInfo` from being included when serialising to JSON, which was causing some integration tests to fail.